autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/CloudbreakCommunicator.java

```diff
@@ -58,7 +58,7 @@ public StackStatusV4Response getStackStatusByCrn(String stackCrn) {
     public void decommissionInstancesForCluster(Cluster cluster, List<String> decommissionNodeIds) {
         ScalingStrategy scalingStrategy = ScalingStrategy.STOPSTART;
         requestLogging.logResponseTime(() -> {
-            if (cluster.isStopStartScalingEnabled()) {
+            if (Boolean.TRUE.equals(cluster.isStopStartScalingEnabled())) {
```

Comment:
@aakashshankar I saw that downscale by stop was getting invoked even if stop-start scaling was disabled. I guess this is the fix for the same. CC: @sidseth

Reply:
I see what is happening. It is unrelated. The cloudbreak core code just checks for the entitlement is enabled or not, and it does not have a way to check whether the cluster has the stop/start flag enabled or not. This will be a big problem for rolling out to mow-dev. CC: @sidseth

Reply:
This needs more investigation from my side.